### ISSUE TITLE
Einheitliche Anführungs- und Schlusszeichensetzung

### ISSUE THREAD

**Issue by vrukavina** [GF-Corporate-Archives/gf-johann-conrad-fischer#482]
Mir ist beim nochmaliger Kontrolle der englischen Tagebücher aufgefallen, dass die Anführungs- und Schlusszeichen bei den neuen en TB mal ausserhalb, mal innerhalb des Tags sind. Weil wir zuvor in en TB 7 die Zeichen immer ausserhalb gesetzt haben, und ich es auch sinnvoll finde, weil mit ihnen bloss auf die originale (englische) Schreibweise Fischers hingewiesen wird, habe ich es bei den neuen TB jetzt auch so vereinheitlicht.
"Ausnahmen" gibt es wenige, bei denen es 'korrekt ist' und es auch schöner aussieht, wenn das in Anführungs- und Schlusszeichen gesetzte Wort Teil des Tags ist, wie z.B. : 
![Bildschirmfoto 2023-10-10 um 10 54 23](https://github.com/GF-Corporate-Archives/gf-johann-conrad-fischer/assets/102786242/ef74e584-e214-4abe-89e6-19771819ea7a) im TB 4
oder im TB 3: 
![Bildschirmfoto 2023-10-10 um 10 57 29](https://github.com/GF-Corporate-Archives/gf-johann-conrad-fischer/assets/102786242/524115fd-52e8-452e-b98d-2c3e45a2b127)

@NicolauLutz Bitte auch bei den englischen TB 5 und TB 6 so vorgehen, um später Zeit zu sparen.

**Comment by feggimann** [comment]
Die Anführungszeichen sollten nicht mitgetaggt sein, das ist richt. Bitte aber im Moment warten mit Anführungszeichen bearbeiten, das scheint eine grössere Baustelle. S. auch Issue #483

**Comment by feggimann** [comment]
Ich werde nächste Woche mit Andreas schauen, ob wir das automatisiert ändern, oder immerhin danach suchen können.

**Comment by vrukavina** [comment]
Ja, Nicolau wird dieses Issue mit Issue #533 gleichzeitig bearbeiten.
